lightning_examples/warp-drive/multi_agent_rl.py

```diff
@@ -1,3 +1,11 @@
+# %%
+import sys
+
+if "google.colab" in sys.modules:
+    ! pip uninstall --quiet "torchaudio" "torchvision" "torchtext" "fastai" "datascience" "kapre" --yes
+    ! pip install --quiet "rl-warp-drive>=1.6.5" "pytorch-lightning>=1.4"
+else:
+    ! pip install --quiet "ipython[notebook]" "rl-warp-drive>=1.6.5" "ffmpeg-python" "setuptools==59.5.0" "torch>=1.8" "torchmetrics>=0.7" "pytorch-lightning>=1.4"
```

Comment:
this is not valid python, if you have some additional comment on how to install it on different env such as colab, lest add it as text celll

Reply:
We could use `subprocess.run(..)`

Reply:
this is not the point, you try to band the system for this specific notebook and specific env of Colab...